app/src/main/res/menu/document__edit__menu.xml

```diff
@@ -152,6 +152,10 @@
                             android:id="@string/action_format_plaintext"
                             android:icon="@drawable/ic_file_white_24dp"
                             android:title="@string/plaintext" />
+                        <item
+                            android:id="@string/action_format_asciidoc"
+                            android:icon="@drawable/asciidoc_icon_black_24dp"
```

Comment:
What is the license and source of the asciidoc icon?   
Preferably we take a icon from the Android material gallery. This means it matches perfectly size/grid wise, anybody can later on regenerate it when something changes on the material design / glyphs, and it's for sure fine license wise.  

A option is for example just making a text-based icon with character "A".

Reply:
This is the new official logo, which has not been around that long. But which can and may be used for it.
For me, it is important, to use the official logo.

https://www.eclipse.org/org/artwork/

https://www.eclipse.org/org/artwork/zip_files/asciidoc-logo.zip

https://gitlab.eclipse.org/eclipse-wg/asciidoc-wg/asciidoc-wg/-/tree/main/brand

AsciiDoc Brand Guidlines
https://www.eclipse.org/org/artwork/guidelines/asciidoc-brand-guidelines.pdf

The question about the usage of icons have been discussed in AsciiDoc groups. If you need, I can explicitely ask for usage.

Reply:
Preferably as said it's something supported by Material Design. It also means it's perfectly tintable and requiring only one vector icon. As the icon is displayed at light and dark backgrounds it needs to be automatically adaptable.

Like settings and toolbar in light and dark app mode usage.

As you can see here, all icons work perfectly fine and they look uniformed. I sure prefer a perfectly working icon with `A` character over a blue trademarked icon just because it's something official.

![screenshot-2022-10-05__19-15-30](https://user-images.githubusercontent.com/6735650/194121508-d7b44500-3410-4e3b-98b1-aa038d5bdcb6.png)

Reply:
Using the AsciiDoc icon is also a form of appreciation. A lot of energy has gone into it.

Equally important is the correct spelling "AsciiDoc", and not "Asciidoc" or "asciidoc".